examples/minimal/packages/client-react/package.json

```diff
@@ -25,7 +25,7 @@
     "@latticexyz/world": "link:../../../../packages/world",
     "@wagmi/chains": "^0.2.22",
     "async-mutex": "^0.4.0",
-    "contracts": "workspace:*",
+    "contracts": "workspace:../contracts",
```

Comment:
any reason this changed?

Reply:
forget why i changed, just tried original and it worked, changing back